src/datadog/tracer_telemetry.cpp

```diff
@@ -241,5 +332,22 @@ std::string TracerTelemetry::app_closing() {
   return message_batch_payload;
 }
 
+std::string TracerTelemetry::configuration_change(
+    const std::vector<ConfigMetadata>& new_configuration) {
+  auto configuration_json = nlohmann::json::array();
+  for (const auto& config_metadata : new_configuration) {
+    // if (config_metadata.value.empty()) continue;
+    configuration_json.emplace_back(
+        generate_configuration_field(config_metadata));
+  }
+
+  auto configuration_change =
+      generate_telemetry_body("app-client-configuration-change");
```

Comment:
Be sure to mention this in the comments at the top of `tracer_telemetry.h`. That is also a good time to audit the code for any telemetry event types that we forgot to mention before.

Reply:
addressed in e2bb8f465b2245055cb150d6d5bc8bcf69211c65